ui/imports/shared/status/StatusStickersPopup.qml

```diff
@@ -275,36 +275,35 @@ Popup {
                 highlighted: true
                 onClicked: {
                     highlighted = true
-                    stickerPackListView.selectedPackId = -1
+                    stickerPackListView.selectedPackId = ""
                     d.getRecentStickers()
                     stickerGrid.model = d.recentStickers
                 }
             }
 
             StatusScrollView {
-                height: 40
                 Layout.fillWidth: true
                 Layout.fillHeight: true
+                leftPadding: 0
+                rightPadding: 0
                 ScrollBar.vertical.policy: ScrollBar.AlwaysOff
 
                 RowLayout {
-                    spacing: Style.current.padding
+                    spacing: footerContent.spacing
 
                     Repeater {
                         id: stickerPackListView
-                        property int selectedPackId: -1
+                        property string selectedPackId
                         model: d.stickerPackList
                         visible: d.stickerPacksLoaded
```

Comment:
I guess it has more sense to have here `model: d.stickerPacksLoaded? d.stickerPackList : []` then setting `visible` prop of `Repeater`.

Reply:
I agree... I'd change so many things here :) Didn't want to spend more time/effort than necessary (this is existing code, just fixed it to make it work)